nuxeo-features/nuxeo-thumbnail/src/test/resources/log4j2-test.xml

```diff
@@ -12,6 +12,7 @@
   </Appenders>
   <Loggers>
     <Logger name="org.nuxeo" level="info" />
+    <Logger name="org.nuxeo.lib.stream" level="debug" />
```

Comment:
do we need to keep this?

Reply:
Yes I put this for debug purpose but now everything is fine so...